docs/src/staking.md

```diff
@@ -68,6 +68,8 @@ Check out our [guide to using SolFlare](wallet-guide/solflare.md) for details.
  Ledger Nano.
  [Staking commands using the Solana Command Line Tools](cli/delegate-stake.md).
 
+ - Exodus makes the process very simple. See their [FAQ](https://support.exodus.com/article/1551-solana-staking-faq) for details.
```

Comment:
I think we should add a note that it's not possible to select a validator to stake to with this wallet. Similar to how Binance and FTX work (which also aren't listed here)

Reply:
Sure. I will update.

Reply:
@jstarry updated,

Reply:
Thanks!